libraries/rush-lib/src/cli/RushXCommandLine.ts

```diff
@@ -189,6 +224,10 @@ export class RushXCommandLine {
           quiet = true;
         } else if (argValue === '-h' || argValue === '--help') {
           help = true;
+        } else if (argValue === '-d' || argValue === '--debug') {
```

Comment:
@octogonz - Why aren't we using ts-command-line here?

Reply:
Seems like `rushx` and `rush`'s command-lines should extend from a common base.

Reply:
Can we make that change in another PR?

Reply:
@iclanton `rushx` intentionally does not use `ts-command-line` because (1) its CLI is extremely simple (in the past at least) and (2) almost all of the CLI parameters are passed through to the wrapped command, which is a feature not very well supported by `ts-command-line` today.



> Can we make that change in another PR?

+1 If we did decide to make such a change, it would need to be a separate PR. It is a major architectural change that is separate from the goals of this PR.